joint-discovery-gatk4.wdl

```diff
@@ -403,42 +420,38 @@ task GenotypeGVCFs {
   String output_vcf_filename
 
   String gatk_path
-  String java_opt
 
   File ref_fasta
   File ref_fasta_index
   File ref_dict
 
   String dbsnp_vcf
-
-  String docker_image
+  String docker
   Int disk_size
-  String mem_size
-  Int preemptibles
 
   command <<<
     set -e
 
     tar -xf ${workspace_tar}
     WORKSPACE=$( basename ${workspace_tar} .tar)
 
-    ${gatk_path} --java-options "${java_opt}" \
+    ${gatk_path} --java-options "-Xmx5g -Xms5g" \
```

Comment:
I really don't want to hard code this because reading from GenomicsDB requires more memory for larger sample sizes.  I'm running tasks right now where I'm giving it 10GB.